src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -5581,25 +5581,32 @@ public static void addItemAdjustment(
   private static void addOffhandRemarkable(
       Map<Slot, AdventureResult> equipment, Modifiers newModifiers) {
     var offhand = equipment.get(Slot.OFFHAND);
-    addOffhandRemarkable(offhand, newModifiers);
+    addOffhandRemarkable(equipment, offhand, newModifiers);
     // and also offhand items that are equipped on the familiar
     var famItem = equipment.get(Slot.FAMILIAR);
-    addOffhandRemarkable(famItem, newModifiers);
+    addOffhandRemarkable(equipment, famItem, newModifiers);
   }
 
-  private static void addOffhandRemarkable(AdventureResult item, Modifiers newModifiers) {
-    if (item != null
-        && item != EquipmentRequest.UNEQUIP
-        && ItemDatabase.getConsumptionType(item.id) == ConsumptionType.OFFHAND) {
-      if (item.id != ItemPool.LATTE_MUG) {
-        var mods = ModifierDatabase.getItemModifiers(item.id);
+  private static void addOffhandRemarkable(
+      Map<Slot, AdventureResult> equipment, AdventureResult item, Modifiers newModifiers) {
+    if (item == null
+        || item == EquipmentRequest.UNEQUIP
+        || ItemDatabase.getConsumptionType(item.id) != ConsumptionType.OFFHAND
+        || item.id == ItemPool.LATTE_MUG) {
```

Comment:
I wasn't able to determine why this is excluded as I don't have a latte mug of my own. Is this intended?

Reply:
yes: mug enchants aren't doubled.